README.md

```diff
@@ -13,19 +13,26 @@ vtebench -w $(tput cols) -h $(tput lines) [-c|-b=BYTES|-t=TERM] <benchmark>
 
 Terminal protocol will be output to `stdout`. Output **must be** directed into a
 file rather than used directly to benchmark. `vtebench` is written for ease of
-understanding, **not** performance. To benchmark the currently running terminal
-then, something like this would work:
+understanding, **not** performance.
 
-```sh
-vtebench -w $(tput cols) -h $(tput lines) alt-screen-random-write \
-    > /tmp/100mb.vte
+To generate the most basic commands, the
+[`generate-benchmarks.sh`](./generate-benchmarks.sh) script can be used. This
+should be run in the projects root directory and will output the benchmark
+files to `target/benchmarks`.
+
+After the files have been generated, the performance can be measured with
+[perf](https://perf.wiki.kernel.org/index.php/Main_Page), or
+[hyperfine](https://github.com/sharkdp/hyperfine) on macOS or Windows:
 
-time cat /tmp/100mb.vte
+```sh
+perf stat -r 10 cat target/benchmarks/alt-screen-random-write.vte
+hyperfine --show-output "cat target/benchmarks/scrolling.vte"
```

Comment:
I still feel like adding an explicit option to run 10 times makes more senes. hyperfine can change default number of runs and it could break our README, and also it'll be consistent with perf visually. Feel free to ignore this one.

Reply:
I'm pretty sure hyperfine is slightly smart about it though, so even if the default changed to something less (let's say 3), it'll still run enough times to give you reliable results.

That's why there's min-runs, max-runs and exact number of runs all as options.

So I think hyperfine kinda encourages to run this with just the defaults and trust what it does, so I feel like that should be what we recommend too as long as it works out.